test/e2e/apps/replica_set.go

```diff
@@ -35,7 +36,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *extensions.ReplicaSet {
+func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *apps.ReplicaSet {
```

Comment:
I'm not sure. Just being curious, why do we have to gate? Is it because `apps/v1beta2` may not be enabled in old e2e clusters?